test/integration/cypress/support/page_files/NodePage.ts

```diff
@@ -7,23 +7,46 @@ class EdgeList {
     return cy.get(this.selector + " [data-cy=edge]");
   }
 
+  get list() {
+    return cy.get(this.selector);
+  }
+
   get title() {
     return cy.get(this.selector + " [data-cy=edge-list-title]");
   }
 }
 
+export enum NodePageTab {
+  PredicateGrid = "predicate-grid",
+  PredicateList = "predicate-list",
+}
+
 export class NodePage extends Page {
   constructor(private readonly nodeId: string) {
     super();
   }
 
+  assertTabSelected(tab: NodePageTab) {
```

Comment:
Conversely, I prefer the assertions be in the spec and not in the page object.
The page object has something like getCurrentTabClass { cy.get(...); }, assuming that's possible.
Feel free to rebut.

Reply:
I put the assertion in here because the class that is being checked is specific to the implementation.  To me, checking that a specific class exists on an element is too specific for the natural language feel we are going for in the tests.

Reply:
For classes it's difficult. You can cy.get(...).invoke("text") to return text contents of an element from a getter, for example. I don't know any equivalent for classes, though there probably is one (.attr?).

In any case, probably not worth the effort to change.